_data/cn/stage3.yml

```diff
@@ -0,0 +1,253 @@
+- title: JavaScript中的旧 RegExp功能
```

Comment:
Do we really need to translate proposals? That requires much more maintain work

Reply:
The churn on this is low -- we usually have 2-3 proposals move every 2 months. I don't know if this will be a lot of work... I am hoping that this first round gets the heavy translation work out of the way and from this point forward it is easier...